src/guessNumber.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const readline = require('readline');
+
+const { getBullsAndCows } = require('./getBullsAndCows');
+const { isNumberValid } = require('./isNumberValid');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const messages = {
+  error: 'Please enter valid number (4 different digits) ',
+  tryAgainMessage: 'Please, try again ',
+  success: 'Congratulation! You did it)',
+};
+
+function guessNumber(secret, message) {
+  terminal.question(message, (guess) => {
+    if (!isNumberValid(guess)) {
+      guessNumber(secret, messages.error);
+
+      return;
+    }
+
+    const { bulls, cows } = getBullsAndCows(secret, guess);
+
+    if (bulls !== 4) {
```

Comment:
What is this number actually mean?
Do you remember why it's equal to 4 after a month?
It would help if you created a specific variable for such cases